docs/library/espnow.rst

```diff
@@ -780,6 +799,7 @@ espnow:
   sta.connect('myssid', 'mypassword')
   while not sta.isconnected():  # Wait until connected...
       time.sleep(0.1)
+  sta.config(pm=sta.PM_NONE)  # ..then disable power saving
   ap.active(True)         # Disable power-saving mode
```

Comment:
Should this line `ap.active(True)` be removed now that the STA can disable power saving?